utils/test.js

```diff
@@ -16,3 +16,89 @@ exports.mockCalendlyAuthentication = () => {
       },
     });
 };
+
+exports.mockCalendlyScheduledEvents = () => {
+  nock(CALENDLY_API_BASE_URL)
+    .persist()
+    .get('/scheduled_events')
+    .reply(200, {
+      resource: {
+        collection: [],
+        pagination: {},
+      },
+    });
+};
```

Comment:
This is something we could definitely add later, but it might be nice to have a way for the mock to return data.
```suggestion
exports.mockCalendlyScheduledEvents = (collection = [], pagination = {}) => {
  nock(CALENDLY_API_BASE_URL)
    .persist()
    .get('/scheduled_events')
    .reply(200, {
      resource: {
        collection: collection,
        pagination: pagination,
      },
    });
};
```

Reply:
Right now, it looks like you're using cypress intercepts instead of having the mock return the data you want, which is also fine.

Reply:
Yeah, I kind of followed Tristan's lead here. Can you elaborate on why it might be good to return data? I guess it's a good added layer, right?

Reply:
You can write your mocks to return the data that your tests need instead of intercepting with cypress. The more heavily you use mocks over intercepts, the more valuable this becomes. Right now, you aren't really using the mocks to mock a reply. They're just preventing the test server from making http calls. Your intercepts are responsible for providing the data. Both ways are fine.

Reply:
Got it! Okay. Yeah, I thought the idea here was to just protect against the test server. I think I may have tried returning data at some point, but not sure. It was a while ago! I'm going to revisit this.